### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by mejo-** [nextcloud/collectives#159]
In GitLab by @azul on Apr 26, 2021, 10:33

Go through the app with a tool such as `gnome screenreader` or `lighthouse` firefox extension and check if the text makes sense if one cannot see the icons.
